cmake/Modules/FindTorch.cmake

```diff
@@ -0,0 +1,75 @@
+# FindTorch
```

Comment:
I'm not a CMake expert. But I think the recommended way is to produce -config.cmake file instead of FindXXX.cmake.

This is how Caffe2 does it: https://github.com/pytorch/pytorch/blob/master/cmake/Caffe2Config.cmake.in https://github.com/pytorch/pytorch/commit/a1518b780138400af98df6ea0096f39f2481191d#diff-dcfb71fad8d9192fd423940106d63b66

Resident CMake experts: @Yangqing @orionr @ezyang

Reply:
Checked with @Yangqing offline. -config.cmake is indeed a recommended way - you can follow the C2 example.

Or see https://gitlab.kitware.com/cmake/community/wikis/doc/tutorials/How-to-create-a-ProjectConfig.cmake-file

Reply:
Ok I'll try this